.github/workflows/publish_website.yml

```diff
@@ -34,8 +34,17 @@ jobs:
         uses: actions/setup-python@7f80679172b057fc5e90d70d197929d454754a5a # v2
         with:
           python-version: 3.x
-      - name: Installing Material for MkDocs
-        run: pip install mkdocs-material
+          cache: 'pip'
+      - name: Installing dependencies
+        run: |
+          pip install \
+            mkdocs-material \
+            requests
+      - name: Generating release index
+        run: |
+          build_tools/scripts/generate_release_index.py \
+            --owner="${GITHUB_REPOSITORY}" \
+            --output=docs/website/overrides/pip-release-links.html
```

Comment:
Did you test that this file gets built/served? (`mkdocs serve`, https://github.com/iree-org/iree/tree/main/docs/website#how-to-edit-this-documentation)

Reply:
I am trying to at the moment and it does not, which I don't really understand because it *seems* like the docs say you can add extra files in overrides...

Reply:
Maybe just drop it in `docs/website/docs/` ?

https://stackoverflow.com/a/47189882 suggests you can use .html files...

Reply:
Yeah that works. I'm still surprised overrides doesn't, but I guess that's only for things that are overriding existing files provided by the theme

Reply:
Did you mean to push a commit with that change?

Reply:
Yes, and now I have :-)